libs/model/src/middleware/authorization.ts

```diff
@@ -125,6 +126,7 @@ export const loadComment: CommentMiddleware = async ({ payload }) => {
   return comment;
 };
 
+// eslint-disable-next-line @typescript-eslint/require-await
 export const isCommentAuthor: CommentMiddleware = async ({ actor }, state) => {
   // super admin is always allowed
   if (actor.user.isAdmin) return;
```

Comment:
I'm trying to break these things down into smaller branches.  I think we SHOULD fix these but for now my primary goal is to just get lint-diff everywhere. I'm totally fine with another PR on top of this one that resolve these though.

Reply:
It's a 1 line change where instead of a comment it's modifying line 132. Doesn't even make the size of the PR grow... If we add these comments we will not know that we need to fix these later.

Reply:
Ah... it's actually possible because we can return Promise.resolve() but note that it was actually two lines :)